$NetBSD: patch-al,v 1.2 2022/06/06 05:11:58 dholland Exp $

- Remove bashisms.
- Fix BLT library names.
- Avoid print from test(1) because ITCL_MAJOR_VERSION is never actually set.

Caution: it appears that not all of the direct changes to configure in
its own patch (patch-aa) are reflected here.

--- configure.in.orig	2004-04-14 03:39:12.000000000 +0000
+++ configure.in
@@ -107,7 +107,7 @@ TK_LIB_STATIC=${TKCONF_PATH}/libtk${TK_V
 # Check for the TkStep libraries (if needed)...
 #
 
-if test "$ENABLE_TKSTEP" == "yes" ; then
+if test "$ENABLE_TKSTEP" = "yes" ; then
   TK_LIB_SPEC="-ltkstep -lXpm -ltiff"
   AC_MSG_NOTICE([will link using TkStep libraries])
 fi
@@ -120,7 +120,7 @@ AC_MSG_CHECKING(version of Tcl)
 AC_MSG_RESULT($TCL_VERSION)
 if test "$TCL_MAJOR_VERSION" -lt 8; then
    echo
-   if test "$ENABLE_UNSUPPORTED" == "yes" ; then
+   if test "$ENABLE_UNSUPPORTED" = "yes" ; then
       AC_MSG_WARN([you may be attempting to use TkDesk 2.x with an unsupported version of Tcl.  Good luck.  Hope it works out for you.])
    else
       AC_MSG_ERROR([TkDesk version 2.x requires Tcl version 8 or higher.  Please upgrade your Tcl or, if that is not possible, you might prefer to use TkDesk version 1.2 instead.])
@@ -131,7 +131,7 @@ AC_MSG_CHECKING(version of Tk)
 AC_MSG_RESULT($TK_VERSION)
 if test "$TK_MAJOR_VERSION" -lt 8; then
    echo
-   if test "$ENABLE_UNSUPPORTED" == "yes" ; then
+   if test "$ENABLE_UNSUPPORTED" = "yes" ; then
       AC_MSG_WARN([you may be attempting to use TkDesk 2.x with an unsupported version of Tk.  Good luck.  Hope it works out for you.])
    else
       AC_MSG_ERROR([TkDesk version 2.x requires Tk version 8 or higher.  Please upgrade your Tk or, if that is not possible, you might prefer to use TkDesk version 1.2 instead.])
@@ -165,9 +165,9 @@ if test $success = 1 ; then
 
    AC_MSG_CHECKING(version of Itcl)
    AC_MSG_RESULT($ITCL_VERSION)
-   if test "$ITCL_MAJOR_VERSION" -ne 3; then
+   if test "${ITCL_MAJOR_VERSION:-3}" -ne 3; then
       echo
-      if test "$ENABLE_UNSUPPORTED" == "yes" ; then
+      if test "$ENABLE_UNSUPPORTED" = "yes" ; then
 	 AC_MSG_WARN([you may be attempting to use TkDesk 2.x with an unsupported version of Itcl.  Good luck.  Hope it works out for you.])
       else
 	 AC_MSG_ERROR([found Itcl, but it appears to be the wrong version.  TkDesk 2.x requires itcl 3.x to be installed.  Please get it from: http://www.incrtcl.sourceforge.net.])
@@ -222,17 +222,17 @@ if test -z "$BLT_LIB_PATH" ; then
    NEED_BLT_LIB=lib_blt
    AC_MSG_RESULT(./blt)
 else
-   BLT_LIB_SPEC="-L$BLT_LIB_PATH -lBLT"
-   BLT_LIB_STATIC=$BLT_LIB_PATH/libBLT.a
+   BLT_LIB_SPEC="-L$BLT_LIB_PATH -lBLT24"
+   BLT_LIB_STATIC=$BLT_LIB_PATH/libBLT24.a
    NEED_BLT_LIB=
-   if test   ! -f $BLT_LIB_PATH/libBLT.a \
-          -a ! -f $BLT_LIB_PATH/libBLT.so
+   if test   ! -f $BLT_LIB_PATH/libBLT24.a \
+          -a ! -f $BLT_LIB_PATH/libBLT24.so
    then
-      if test   ! -f $BLT_LIB_PATH/lib/libBLT.a \
-	     -a ! -f $BLT_LIB_PATH/lib/libBLT.so
+      if test   ! -f $BLT_LIB_PATH/lib/libBLT24.a \
+	     -a ! -f $BLT_LIB_PATH/lib/libBLT24.so
       then
 	 echo
-	 AC_MSG_ERROR([could not find libBLT.a or libBLT.so in $BLT_LIB_PATH or in $BLT_LIB_PATH/lib.])
+	 AC_MSG_ERROR([could not find libBLT24.a or libBLT24.so in $BLT_LIB_PATH or in $BLT_LIB_PATH/lib.])
       else
 	 BLT_LIB_PATH=$BLT_LIB_PATH/lib
       fi
